JPS_Ontology/ontology/ontoassetmanagement/OntoAssetManagement.csv

```diff
@@ -30,8 +30,38 @@ UnitPrice,Class,Price,IS-A,,,,Price per individual product or service.,https://w
+Event,Class,,,,,,"A thing that happens or takes place, especially one of importance.",https://www.theworldavatar.com/kg/ontoassetmanagement/,Event
+AssetUpdateHistory,Class,,,,,,A record of updates applied to the information of an asset.,https://www.theworldavatar.com/kg/ontoassetmanagement/,Asset update history
+Furniture,Class,,,,,,"Objects intended to support various human activities such as seating (e.g., stools, chairs, and sofas ), eating ( tables ), storing items, working, and sleeping (e.g., beds and hammocks ). ",https://www.theworldavatar.com/kg/ontoassetmanagement/,Furniture
+Cabinet,Class,Furniture,IS-A,,,,A case or cupboard with shelves and/or drawers for storing or displaying items.,https://www.theworldavatar.com/kg/ontoassetmanagement/,Cabinet
+Project,Class,,,,,,"Any undertaking, carried out individually or collaboratively and possibly involving research or design, that is carefully planned to achieve a particular goal.",https://www.theworldavatar.com/kg/ontoassetmanagement/,Project
+ResearchGrant,Class,,,,,, A grant obtained for conducting scientific research generally through a competitive process.,https://www.theworldavatar.com/kg/ontoassetmanagement/,Research grant
+Budget,Class,,,,,,The amount of money needed or available for a purpose. ,https://www.theworldavatar.com/kg/ontoassetmanagement/,Budget
+BudgetCategory,Class,,,,,,A sub-divison of a budget allocated for a more specific purpose.,https://www.theworldavatar.com/kg/ontoassetmanagement/,Budget category
+ServiceCode,Class,,,,,,An attribute that describes the service code of a budget category.,https://www.theworldavatar.com/kg/ontoassetmanagement/,Service code
+ServiceCategory,Class,,,,,,An attribute that describes the service category of a budget category.,https://www.theworldavatar.com/kg/ontoassetmanagement/,Service category
+lastAssignedTo,Object Property,,,https://w3id.org/bot#Element,https://spec.edmcouncil.org/fibo/ontology/FND/Parties/Parties/IndependentParty,,A relationship which describes who the asset was last assigned to.,https://www.theworldavatar.com/kg/ontoassetmanagement/,last assigned to
+wasExecutedBy,Object Property,,,Event,https://spec.edmcouncil.org/fibo/ontology/FND/Parties/Parties/IndependentParty,,A relationship which describes the party that executed the event.,https://www.theworldavatar.com/kg/ontoassetmanagement/,was executed by
+hasEvent,Object Property,,,AssetUpdateHistory,Event,,A relationship which describes the events that has occurred.,https://www.theworldavatar.com/kg/ontoassetmanagement/,has event
+hasUpdateHistory,Object Property,,,https://w3id.org/bot#Element,AssetUpdateHistory,,A relationship that describes the update history of something.,https://www.theworldavatar.com/kg/ontoassetmanagement/,has update history
+allocatedUnder,Object Property,,,https://w3id.org/bot#Element,Project,,A relationship which describes the project that an asset is allocated under.,https://www.theworldavatar.com/kg/ontoassetmanagement/,allocated under
```

Comment:
Sounds odd to me. Isn't something rather allocated to sth?

Reply:
Now that you mention it, yes, it sounds weird. I will change it to allocatedTo.